src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,12 +1,7 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
-    public Bulldozer() {
-    }
+    private int size = 0;
```

Comment:
Also size = 0 is redundant

Reply:
![image](https://user-images.githubusercontent.com/79330731/195574556-b157cd75-b641-453f-96d1-7cf0060f229e.png)